layouts/get-almalinux/single.html

```diff
@@ -1410,13 +1410,17 @@ <h2 class="pb-2 text-center border-bottom" style="border-bottom-color: #14598a!i
                     </p>
                     <div class="AL" style="display: flex;">
                       <div class="itemAl_01" style="width: 200px;">
-                        <b><a href="https://repo.almalinux.org/rpi/9/images/">AlmaLinux OS 9.5</a></b>
+                        <b><a href="https://repo.almalinux.org/almalinux/10/raspberrypi/images/">AlmaLinux OS 10.0</a></b>
                       </div>
-                      <div class="itemAl_02"  style="margin-left: 100px;">
-                        <b><a href="https://repo.almalinux.org/rpi/images/">AlmaLinux OS 8.10</a></b>
+                      <div class="itemAl_02" style="width: 200px; margin-left: 100px;">
+                        <b><a href="https://repo.almalinux.org/almalinux/9/raspberrypi/images/">AlmaLinux OS 9.6</a></b>
+                      </div>
+                      <div class="itemAl_03" style="margin-left: 100px;">
+                        <b><a href="https://repo.almalinux.org/almalinux/8/raspberrypi/images/">AlmaLinux OS 8.10</a></b>
                       </div>
                     </div>
                     <p style="margin-bottom: 0px;">
+                      {{ i18n "Default Login" }}: almalinux / almalinux (username/password) <br>
                       {{ i18n "More details can be found on the" }} 
```

Comment:
What about "More details such as default login can be found on the...." instead?